src/core/agent/services/multiSigService.test.ts

```diff
@@ -1982,4 +2039,169 @@ describe("Multisig sig service of agent", () => {
     });
     expect(sendExchangesMock).toBeCalledTimes(1);
   });
+
+  test("should can call multisigAdmit when accept ACDC from issuer with route ipex/grant", async () => {
+    const multisigSignifyName = "multisigSignifyName";
+    const notificationSaid = "ELykd_2bX6yvuVEgLQqnCgZ7QLdxpUBze-RzHVwfCUfW";
+    const mockAdmit = {
+      _kind: "JSON",
+      _raw: "{\"v\":\"KERI10JSON000111_\",\"t\":\"exn\",\"d\":\"EKlXGS9FrnHFWvo1HB2KsyXbXDRVts42nKg7qmzKas0j\",\"i\":\"EFKiC-cDUpBauOHMLRRKGaDBZue2FCqTrJJU8nn8qv_A\",\"p\":\"EGBtDLn59jI97a_Y0Poztc4wM3SZdTyjW8gRcBiMNuEy\",\"dt\":\"2024-08-02T02:47:15.250000+00:00\",\"r\":\"/ipex/admit\",\"q\":{},\"a\":{\"m\":\"\"},\"e\":{}}",
+      _ked: {
+        v: "KERI10JSON000111_",
+        t: "exn",
+        d: "EKlXGS9FrnHFWvo1HB2KsyXbXDRVts42nKg7qmzKas0j",
+        i: "EFKiC-cDUpBauOHMLRRKGaDBZue2FCqTrJJU8nn8qv_A",
+        p: "EGBtDLn59jI97a_Y0Poztc4wM3SZdTyjW8gRcBiMNuEy",
+        dt: "2024-08-02T02:47:15.250000+00:00",
+        r: "/ipex/admit",
+        q: {},
+        a: { m: "" },
+        e: {},
+      },
+      _size: 273,
+    };
+    const mockSigs = [
+      "AAAWesHyXn8p_ZeQYxD52OXhYkZiDNNmNBq8Bat_n7wk0KZ1LXKpfpeOd9s3A2huVibe1J18AgI-NOH9lfnvEOAM",
+    ];
+    const mockEnd = "";
+
+    mockResolveOobi.mockResolvedValueOnce({
+      name: "oobi.AM3es3rJ201QzbzYuclUipYzgzysegLeQsjRqykNrmwC",
+      metadata: {
+        oobi: "testOobi",
+      },
+      done: true,
+      error: null,
+      response: {},
+      alias: "c5dd639c-d875-4f9f-97e5-ed5c5fdbbeb1",
+    });
+
+    getExchangesMock.mockResolvedValueOnce({
+      exn: {
+        d: "EO65SZOen5Qm26gYeAZZ_J_p8_Uy_6jB3cUpv0DzgDA4",
+      },
+    });
+    identifiersMemberMock = jest.fn().mockResolvedValueOnce({
+      signing: [{ ends: { agent: { [keriMetadataRecord.id]: "" } } }],
+    });
+    identifierStorage.getIdentifierMetadata = jest.fn().mockResolvedValueOnce(
+      new IdentifierMetadataRecord({
+        id: "aidHere",
+        displayName: "Identifier 2",
+        signifyName: "uuid-here",
+        createdAt: now,
+        theme: 0,
+        groupMetadata: {
+          groupId: "group-id",
+          groupInitiator: true,
+          groupCreated: true,
+        },
+      })
+    );
+    identifiersGetMock = jest
+      .fn()
+      .mockResolvedValueOnce(gHab)
+      .mockResolvedValueOnce(mHab);
+    ipexAdmitMock.mockResolvedValue([mockAdmit, mockSigs, mockEnd]);
+    createExchangeMessageMock.mockResolvedValue([
+      mockExn,
+      mockSigsMes,
+      mockDtime,
+    ]);
+
+    await multiSigService.multisigAdmit(multisigSignifyName, notificationSaid);
+    expect(ipexAdmitMock).toBeCalledTimes(1);
+    expect(createExchangeMessageMock).toBeCalledTimes(1);
+    expect(ipexSubmitAdmitMock).toBeCalledTimes(1);
+  });
+
+  test("should can call multisigAdmit when accept ACDC from multisig/exn", async () => {
```

Comment:
"should" or "can" instead of "should can" :)

also:
`can agree to admit a credential with a multi-sig identifier`